src/components/ScrollToTop/ScrollToTop.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState, useEffect } from 'react';
+import styles from './ScrollToTop.module.scss';
+import ArrowUp from '@assets/arrow-up.svg?react';
+
+const ScrollToTop = () => {
+  const [isVisible, setIsVisible] = useState(false);
+
+  // Проверяем позицию скролла
+  const toggleVisibility = () => {
+    if (window.scrollY > 700) {
+      setIsVisible(true);
+    } else {
+      setIsVisible(false);
+    }
```

Comment:
```suggestion
    setIsVisible(window.scrollY > 700)
```

и функции лучше обернуть в useCallback

Reply:
сделано